bundles/org.openhab.binding.wled/src/main/java/org/openhab/binding/wled/internal/WLedHandler.java

```diff
@@ -328,8 +332,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         // Google sends this when it wants white
                         sendWhite();
                     } else {
-                        sendGetRequest("/win&TT=1000&FX=0&CY=0&CL=" + createColorHex(primaryColor) + "&A="
-                                + masterBrightness255);
+                        if (config.segmentIndex == -1) {
```

Comment:
Should this same fix be applied to both full strings and segments, any reason to change what is sent based on the segmentIndex?

I plan on adding JSON API and also removing channels that don't make sense when the segment is set in the near future. So if you have any ideas or wish list feel free to raise them.

Reply:
I'm not aware of the context but using two code pieces to parse the HTTP responses sound a bit fishy.

Reply:
Hi,
i did not include it for segments since it is not clear how master brightness works on those. Will it apply to to the segment or the whole strip? Since I did't want to mess with the Segments for People using it I only applied the change to non segments. There is a new API parameter for segment brightness  since 0.10.0 "&SB=". But in Tests this failed completely so I fell back to previous behaviour.